crt_portal/static/sass/custom/form.scss

```diff
@@ -8,20 +8,12 @@ form#report-form,
 form#filters-form {
   .question--header {
     width: 100%; // IE11 fix
-    // margin-top: 0.5rem;
-    // margin-bottom: units($spacer-3x);
-    padding-top: 32px;
     margin: 0;
-    + .help-text,
-    + .help-text__small,
-    + p {
-      // margin-top: -1rem; // to match other help text margins
```

Comment:
This shit was stuff I left in when I was doing the spacing refactor, and should have been removed ages ago. My bad!